src/js/entities-service/entities/patients.js

```diff
@@ -11,20 +11,18 @@ const TYPE = 'patients';
 const _Model = BaseModel.extend({
   type: TYPE,
   urlRoot: '/api/patients',
-  validate(attrs, { preSave }) {
-    if (preSave) {
-      return this.preSaveValidate(attrs);
-    }
+  /* eslint-disable complexity */
+  validate({ first_name, last_name, birth_date, sex, _groups }) {
```

Comment:
I'm still not wildly satisfied with fe error handling..  it's still possible the preSave was a better idea..  but moving some things around made that a little more complex..